File: DailyFlash Solutions/DailyFlash Solutions/58-DailyFlash_Solutions/27_Mar_Solution_One/Python/Program0.py

l = int(input("Enter length of series  :\t"))
x = int(input("Enter Value of X  :\t"))
y = int(input("Enter Value of Y  :\t"))

lst = []
cnt = 0
t = x+y

for s in range(l) :

	if s%2 != 0 :
		cnt = cnt + (s/t**x)
	else :
		cnt = cnt + (s/t**y)

print("Sum of ",l," elements of series is ",round(cnt,4))

File: DailyFlash Solutions/DailyFlash Solutions/58-DailyFlash_Solutions/27_Mar_Solution_One/C/Program0.c
#include <stdio.h>
#include <math.h>

void main() {

	int l = 0,x = 0,y = 0;
	float sum = 0;
	printf("Enter The Length of the Series\n");
	scanf("%d",&l);

	printf("Enter value of x\n");
	scanf("%d",&x);

	printf("Enter value of y\n");
	scanf("%d",&y);

	int t = x + y;
	for(int i = 1; i <= l; i++){

		if(i%2 == 0)	
			sum+=(i/(pow(t,y)));
		else
			sum+=(i/(pow(t,x)));
	}

	printf("Sum of Series = %.4f\n",sum);
}
